### ISSUE TITLE
v0.8.1 App info as an option

### ISSUE THREAD

**Issue by bilogic** [quaap/LaunchTime#70]
Hi,

When an app's icon is dragged, 2 options appear

- Uninstall
- Shortcut

Possible to add 1 more which is App info? This is for adjustment of notifications, permissions etc. Thank you.

**Comment by quaap** [comment]
I just finished up this feature a few days ago :)

**Comment by quaap** [comment]
Oh, if you are not on Android 7.1+, you'll need to enable "Action Menus" in LaunchTime settings.

**Comment by bilogic** [comment]
Yea got it, thanks!

**Comment by bilogic** [comment]
Sorry, I'm curious, any reason why a different interface?
Why not combine the ```Uninstall/Shortcut``` into the action menus or combine ```Appinfo``` to the existing ```Uninstall/Shortcut```

**Comment by quaap** [comment]
"App actions" as a concept was introduced in Android 7.1, which LaunchTime does make use of on Android 7.1+, so something like Gmail will have quick actions like "Compose" or maybe a commonly used contact name. So when I went to add new actions, the action menu is the logical place as opposed to the crowded target area idea.

On lower versions, I've disabled action menus by default because they wouldn't have any other actions, and people are already used to the uninstall and shortcut targets where they are.

I did consider combining the uninstall and shortcuts into the action menu and enabling it for all versions, but I thought the sudden appearance of "action menus" on long presses might confuse or annoy someone on Android 5.1 or 6, where that's not a concept and wouldn't have much other use. 

I may yet do it, though.  Or make it user-configurable.
